Tasks/Common/nuget-task-common/NuGetToolRunner2.ts

```diff
@@ -98,14 +98,36 @@ export class NuGetToolRunner2 extends ToolRunner {
     public execSync(options?: IExecOptions): IExecSyncResult {
         options = options || <IExecOptions>{};
         options.env = prepareNuGetExeEnvironment(options.env || process.env, this.settings, this.authInfo);
-        return super.execSync(options);
+        let execResult = super.execSync(options);
+        if (execResult.code !== 0) {
+            this._logExecResults(execResult.code, execResult.stderr);
+        }
+        return execResult;
     }
 
     public exec(options?: IExecOptions): Q.Promise<number> {
         options = options || <IExecOptions>{};
         options.env = prepareNuGetExeEnvironment(options.env || process.env, this.settings, this.authInfo);
         return super.exec(options);
     }
+
+
+    private _logExecResults(exitCode: number, stderr: string){
+        try{
+            console.log("##vso[telemetry.publish area=Packaging;feature=NuGetCommand]%s",
+            JSON.stringify({
+                'SYSTEM_JOBID': tl.getVariable('SYSTEM_JOBID'),
+                'SYSTEM_PLANID': tl.getVariable('SYSTEM_PLANID'),
+                'SYSTEM_COLLECTIONID': tl.getVariable('SYSTEM_COLLECTIONID'),
+                'command': tl.getInput("command"),
+                'arguments': tl.getInput("arguments"),
```

Comment:
This will only be helpful if the customer chose "custom". I say we should add the other relevant fields, or remove this.

Reply:
This is a _terse_ set of values that give you _some_ context should the forked program (i.e. nuget) fail.  If you want more you need to look at the log at the beginning of the task.

Reply:
I guess what I'm saying is that this piece of code gives a false sense of telemetry to the developer since arguments is a fields that it's barely ever used in the NuGet task. I'd rather remove it and force the person to look for the corresponding beginning of the task.